Documentation/Releases.md

```diff
@@ -0,0 +1,103 @@
+# How to create swift-testing releases
+
+This document describes how to create a new release of swift-testing using git
+tags.
+
+> [!IMPORTANT]
+> You must have administrator privileges to create a new release in this
+> repository.
+
+## Version numbering
+
+swift-testing uses [semantic versioning](https://semver.org) numbers for its 
+open source releases. We use git _tags_ to publish new releases; we do not use
+the GitHub [releases](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases)
+feature.
+
+At this time, all swift-testing releases are experimental, so the major version
+is always `0`. We are not using the patch component, so it is usually (if not
+always) `0`. The minor component should be incremented by one for each release.
+
+For example, if the current release is version `0.1.0` and you are publishing
+the next release, it should be `0.2.0`.
+
+> [!NOTE]
+> Where you see `x.y.z` in this document, substitute the semantic version you
+> are deploying.
+
+## Creating a branch for the release
+
+Before a release can be published, a branch must be created so that the
+repository can be configured correctly. Ensure any local changes have been saved
+and cleared from the repository (e.g. with `git stash` or `git reset --hard`),
+then run the following commands from within the repository's root directory:
+
+```sh
+git checkout main # or other branch as appropriate
+git pull
+git checkout -b release/x.y.z
+```
+
+## Preparing the repository's contents
+
+The package manifest files (Package.swift _and_ Package@swift-5.11.swift) must
+be updated so that the release can be used as a package dependency:
+
+0. Take note of any availability definitions (passed to the Swift compiler with
```

Comment:
The number 0. seems to cause GitHub to actually use zero as the first number in the list, is that what you intended?